src/Ball.cpp

```diff
@@ -15,7 +15,8 @@ const int VELOCITY_INCREMENT = 2;
 Ball::Ball() {}
 Ball::~Ball() {}
 
-void Ball::init(int x, int y) {
+//void Ball::init(int x, int y) {
```

Comment:
this class has a lot of commented out lines - should clean up before merging this in

Reply:
also @yct4 remind me again why we commented this out?

Reply:
ah nvm i see it below